test/integration/telemetry/test.sh

```
@@ -0,0 +1,102 @@
+
+  # Run some distributed tracing requests
+  oha -n 100 -c 5 http://localhost:3000/api/test?downstream=true
+
+  # Generate some errors
+  oha -n 50 -c 2 http://localhost:3000/api/error || true
+else
+  echo -e "${YELLOW}oha not found, using curl for basic testing${NC}"
+  for i in {1..100}; do
+    curl -s http://localhost:3000/api/test > /dev/null
+  done
+  for i in {1..20}; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)